lib/foreman_maintain/concerns/base_database.rb

```diff
@@ -4,11 +4,21 @@ module BaseDatabase
       def data_dir
         if el7? && package_manager.installed?('rh-postgresql12-postgresql-server-syspaths')
           '/var/opt/rh/rh-postgresql12/lib/pgsql/data/'
+        elsif debian?
+          deb_psql_data_dir
         else
           '/var/lib/pgsql/data/'
         end
       end
 
+      def deb_psql_data_dir
```

Comment:
Is this good or should I query the package, grep the version, trim it to two digits and then use it in the path?

Reply:
Given the config path in etc also contains the version, I'd go with "extract version from pkg and user"

Keep in mind, on Debian PostgreSQL versions are co-installable, so you can have 11 and 12 and 13 on one machine (and idk if there is a good way to learn from the installer answers which one is the "right" one).